color-guide/readme.md

```diff
@@ -0,0 +1,48 @@
+## Let's talk about colours
```

Comment:
Every guide has to state a goal in a blockquote in the beginning, see other guides. What is the reader able to do after reading this?